.travis.yml

```diff
@@ -32,7 +32,7 @@ jobs:
       julia: 1.4
       os: linux
       script:
-        - julia --project --color=yes --check-bounds=yes -e 'using Pkg; Pkg.develop(PackageSpec(name="NeuralVerification", url="https://github.com/JuliaReach/NeuralVerification.jl")); Pkg.instantiate()';
+        - julia --project --color=yes --check-bounds=yes -e 'using Pkg; Pkg.instantiate()';
```

Comment:
we may still need to pass the url. i dont think that the package registered?

Reply:
It looks like the error is because the version, it may be registered

Reply:
It looks like it isn't registered :smile: